docs/packages/pkg/thrift.rst

```diff
@@ -14,11 +18,21 @@ thrift
 -  `Example <https://github.com/ruslo/hunter/blob/master/examples/thrift/CMakeLists.txt>`__
 -  Added by `isaachier <https://github.com/isaachier>`__ (`pr-1064 <https://github.com/ruslo/hunter/pull/1064>`__)
 
+This package **does not** compile the Thrift compiler by default. Nor does it
+compile the `thriftz` and `thrifnb` libraries. It just builds the basic `thrift`
+library, without SSL support. To compile the Thrift compiler, you must pass in
+custom CMake arguments in your toolchain, namely `BUILD_COMPILER=ON`. Similarly,
+to build `thriftz`, pass `WITH_ZLIB=ON`. To build `thriftnb`, pass
+`WITH_LIBEVENT=ON`. To compile with SSL support, pass `WITH_OPENSSL=ON`.
+
+Note, compiling the Thrift compiler requires bison, which will make this library
```

Comment:
Effectively it will make library unusable. There is no such feature as "dynamically cacheable" package. For now package is cacheable or not.

Reply:
So should I just remove the note or am I forced to keep it uncacheable?

Reply:
It's up to you. You can keep uncacheable and add a note that some configurations are not valid. Or remove uncacheable and allow any configurations. I think it make sense to resolve relocation issues of bison.